src/components/Dialogs/ImageInfo/ImageInfoDialogComponent.scss

```diff
@@ -0,0 +1,38 @@
+.image-info-dialog {
+    min-width: 60%;
+    height: 100%;
+    user-select: none;
+
+    .bp3-dialog-body {
+        height: 100%;
+        overflow: hidden;
+        margin-top: 10px;
+        margin-left: 25px;
+
+        .bp3-input-group {
+            margin: 10px 5px 5px;
+        }
+
+        .image-info-panes {
+            display: flex;
+            height: 100%;
+            
+            .image-info-pane {
+                width: 95%;
```

Comment:
I don't think these margins are correct. Rather just use a (much smaller) fixed margin and use CSS `calc` to work out the width. I'm not sure why we need a 40px `margin-bottom`.

Reply:
The 40px margin-bottom is for fully seeing the last line of headers information.